### ISSUE TITLE
Typo in man page

### ISSUE THREAD

**Issue by sebastinas** [pwmt/zathura#73]
*On GitLab by @anntzer on Aug 24, 2018, 15:39*

---

The man page currently reads
```
It is enough to pass the arguments to synctex view's -i option to zathura via --syntex-forward...
```
"--syntex-forward" should be "--synctex-forward".

**Comment by sebastinas** [comment]
mentioned in commit 9771cf1e8e43e08fbc8c8e70988dbb1d409758bb

**Comment by sebastinas** [comment]
Thanks, fixed in 9771cf1e.

**Comment by sebastinas** [comment]
closed
